### ISSUE TITLE
setup test plan for GUI tests

### ISSUE THREAD

**Issue by plata** [PhoenicisOrg/POL-POM-5#1202]
* JavaFX can be tested with TestFX
* which parts of the GUI shall be tested?
* how can we ensure that the GUI works without having to update the test for every small change?

**Comment by plata** [comment]
Do you think this would help us: https://github.com/marketplace/testquality?

**Comment by madoar** [comment]
I'm not entirely sure how `testquality` works.
Does it bind the test plans to github issues?
If this is the case we may have a problem, because I believe we should have a test plan or multiple ones, which we use for each release and I don't want to recreate them.

**Comment by plata** [comment]
From my understanding it rather links issues to broken test cases. However, I'm not 100% sure how it works and if it would help us.
The only thing I could suggest is to try it and then decide if it makes sense for us.

**Comment by madoar** [comment]
Yes, I also believe the easiest way to see if this is a good solution for us, is by testing it.

**Comment by madoar** [comment]
Can you add it to the project?
If it doesn't fulfill our needs, we can remove it later on.

**Comment by plata** [comment]
Mh. Either I'm too stupid or the integration is really strange. It takes me to a form where I shall create a TestQuality site. This requires a password etc. If this is really required, I'm not sure we want this (how shall we manage this?).

**Comment by madoar** [comment]
They don't support authentication via GitHub?
